### DISCUSSION TITLE
[6/29/2023] New Mojo Release!!

### DISCUSSION THREAD

**Discussion by goldiegadde** [modular/modular#418] [Announcements]
Hey everyone 👋 , 

We just updated Playground with a new Mojo🔥 release! To see what's new, [check out the changelog](https://docs.modular.com/mojo/changelog.html).
This release enables notebook sharing in the Playground, feel free to check it out and share the amazing things you are able to do with Mojo 🔥. The [changelog](https://docs.modular.com/mojo/changelog.html#section) describes how to use sharing. 

As always, if you have an active session in the Mojo Playground, you'll need to restart it to get the updated build.

Thanks, 
Goldie

**Comment by JUNIOR13-G** [comment]
hi how are you doing?
